ENIGMAsystem/SHELL/Widget_Systems/Cocoa/dialogs.h

```diff
@@ -0,0 +1,41 @@
+/** Copyright (C) 2019 Samuel Venable
+***
+*** This file is a part of the ENIGMA Development Environment.
+***
+*** ENIGMA is free software: you can redistribute it and/or modify it under the
+*** terms of the GNU General Public License as published by the Free Software
+*** Foundation, version 3 of the license or any later version.
+***
+*** This application and its source code is distributed AS-IS, WITHOUT ANY
+*** WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+*** FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+*** details.
+***
+*** You should have received a copy of the GNU General Public License along
+*** with this code. If not, see <http://www.gnu.org/licenses/>
+**/
+
+#include <string>
+
+//  void show_error(std::string errortext, const bool fatal);
+
+namespace enigma_user {
+  
+//  int show_message(std::string str);
+bool show_question(std::string str);
```

Comment:
These declarations almost certainly exist somewhere. They seem to be the same as the declarations in other systems. Are they not in a header in Widget_Systems/General? Or are we not that far along?

Reply:
I can't put it where Windows Widgets' functions Are declared because that system doesn't have all the same functions. Is it ok if we address this another time?

Reply:
I'm surprised we're that far back in the widget systems, but if that's the case, then yes, there's no sense fixing it in this PR.